internal/dag/builder.go

```diff
@@ -518,6 +521,27 @@ func (b *Builder) computeHTTPProxy(proxy *projcontour.HTTPProxy) {
 			svhost.Secret = sec
 			svhost.MinProtoVersion = annotation.MinProtoVersion(proxy.Spec.VirtualHost.TLS.MinimumProtocolVersion)
 
+			// Check if FallbackCertificate && ClientValidation are both enabled in the same vhost
+			if proxy.Spec.VirtualHost.TLS.EnableFallbackCertificate && tls.ClientValidation != nil {
+				sw.SetInvalid("Spec.Virtualhost.TLS.FallbackCertificate & Spec.Virtualhost.TLS.ClientValidation are incompatible together")
```

Comment:
```suggestion
				sw.SetInvalid("Spec.Virtualhost.TLS.EnableFallbackCertificate & Spec.Virtualhost.TLS.ClientValidation are incompatible together")
```

Reply:
I changed to this to be consistent with the requests earlier: `Spec.Virtualhost.TLS fallback & client validation are incompatible together`

Reply:
Nice.